content/docs/04.5-devx/05-app-profile/10-service-listings/3-redis-db.md

```diff
@@ -0,0 +1,112 @@
+
+|         Parameter           | Description  |
+|-----------------------------|-----------------|
+|Application Profile Name | A custom name for the App Profile|
+|Version (optional)       | The default value is 1.0.0. You can create multiple versions of an App Profile using the format **`major.minor.patch`**.
+|Description (optional)   | Description of the App Profile. | 
+|Tag (optional)           | Tags on a cluster group are propagated to the cloud/datacenter environments.|
+ 
+
+4. From the available services start configuring your App Profile. Refer to [App Profiles](/devx/app-profile) for a list of available services.
+
+
+5. Click on **Redis DB** from the DB services and start the configuration.
+  
+
+6. Provide the following information to the wizard:
+  * **Name:** The DB name. You can have the default Palette generated name or create a custom name. 
+  * **Password:** Security password for the DB service.
+
+<InfoBox>
+You can use the default system-generated password. If the default password is used, it can be retrieved from the Redis DB secrets using the following command:
+
+```
+kubectl get secrets -A
+```
+
+For using a custom password, use the [base 64 encoder](https://www.base64encode.org/) to generate an encoded password and add to the basic information wizard. 
+</InfoBox>
+
+  * **Database Volume Size (GiB):** Select the volume as per the storage volume available in the cluster group and virtual clusters. 
+
+6. **Output Variables**: The output variables of this tier that may be used in higher tiers, typically for connection purposes are:
+
+```
+{{.spectro.app.$appDeploymentName.redis-1.USERNAME}}
+```
+```
+{{.spectro.app.$appDeploymentName.redis-1.PASSWORD}}
+```
+```
+{{.spectro.app.$appDeploymentName.redis-1.REDISMSTR_SVC}}
+```
+```
+{{.spectro.app.$appDeploymentName.redis-1.REDISMSTR_SVC_PORT}}
+```
+```
+{{.spectro.app.$appDeploymentName.redis-1.REDISMSTR_NS}}
+```
+
+
+|**Output Variable**|**Description**|
+|---------------|-----------|
+|Username|Username for database access control|
+|Password|Password for database access control|
+|REDISMSTR_SVC|Provides the Redis service fully qualified domain name (FQDN) which can be consumed by App Services for database connectivity|
+|REDISMSTR_SVC_PORT|Represents the port on which the database service is listening to|
+|REDISMSTR_SVC_NAMESPACE|Represents the namespaces to which Redis database is launched|
+
+
+# Validation
+
+* To validate your database service in App Profile, navigate to the **App Profiles** page, where all your app profiles are listed. Click the **App Profile Name** to see the service layers.
+
+
+* Validate the services from the App page after app deployment. First, navigate to the **App** page, where all your apps are listed. Then, click the **App Name** to see the service layers. The color code in the app profile box shows the status of the service deployment.
```

Comment:
```suggestion
* To validate Postgres successfully deployed, navigate to left **Main Menu** and click on **Apps**.  Next, click on the app that has Redits. The following page displays the status of the application and each of its services. The color code in the app profile box shows the status of the service deployment.
```

Reply:
Line 99 suggestion: typo

Next, click on the app that has Redis. The table lists the status of the application and each of its services.